peribolos-fix/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:alpine as builder
+
+RUN apk update && apk add patch curl
+WORKDIR /peribolos
+# As of now the last commit, not downloading the latest becuase the patch might not work
+RUN curl -L https://github.com/kubernetes/test-infra/tarball/83bf07dd99d23a5d6998490ec35240a4be895254 | tar -xz
+WORKDIR /peribolos/kubernetes-test-infra-83bf07d/prow/cmd/peribolos
+RUN go get -d -v
+COPY peribolos.patch .
+RUN patch -f -u main.go -i peribolos.patch
+RUN go build -o /usr/bin/peribolos
+
+FROM alpine
+
+COPY --from=builder /usr/bin/peribolos /usr/bin/peribolos
+CMD /bin/sh
```

Comment:
No need for CMD if you're not changing it to something else than in the base image:

```
$ podman inspect -f 'Entrypoint: {{.Config.Entrypoint}}\nCmd: {{.Config.Cmd}}' alpine:latest
Entrypoint: []
Cmd: [/bin/sh]
$ podman inspect -f 'Entrypoint: {{.Config.Entrypoint}}\nCmd: {{.Config.Cmd}}' registry.access.redhat.com/ubi8-micro:latest
Entrypoint: []
Cmd: [/bin/sh -c /bin/sh]
```